PostgreSQL/test.js

```diff
@@ -0,0 +1,57 @@
+const { getLogger } = require('log4js');
+
+let logger = getLogger(`[MySQL]`);
+logger.level = "TRACE"
+global.logger = logger;
+
+(async () => {
+	let connectorData = {
+		user: 'postgres',
+		host: '0.0.0.0',
+		database: 'postgres',
```

Comment:
Database name used for testing should be dnio-marketpalce

Reply:
Fixed.